automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/SignInPage.scala

```diff
@@ -65,15 +67,28 @@ class SignInPage(val baseUrl: String)(implicit webDriver: WebDriver) extends Fir
 class GoogleSignInPopup(implicit webDriver: WebDriver) extends WebBrowser with WebBrowserUtil {
 
   def awaitLoaded(): GoogleSignInPopup = {
-    await condition (text("to continue to").findElement.isDefined || text("...").findElement.isDefined)
+
+    val popupTrial = Try {
+      await condition  (id("identifierLink").findElement.isDefined || (id("identifierId").findElement.isDefined))
+    }
+
+    popupTrial match {
+      case Success(_) =>
+        val chooseAccountLink = find(id("identifierLink")).isDefined
+        if (chooseAccountLink) {
+          click on (id("identifierLink"))
+          await visible id("identifierId")
+        }
+      case Failure(t) => throw new TimeoutException("Timed out (30 seconds) waiting for Google SignIn Popup.", t)
+    }
+
```

Comment:
rework `GoogleSignInPopup` algorithm. sometimes test failed with screenshot of Google SignIn Popup.